servers/src/main/java/tachyon/master/file/meta/InodeFile.java

```diff
@@ -43,6 +43,7 @@
   private boolean mIsComplete = false;
   private boolean mCache = false;
   private String mUfsPath = "";
+  private long mTTL;
```

Comment:
probably not, right? I can image one want to change the TTL of a file/dir after it is created with an initial TTL value.